### PR TITLE
✨ sidebar filters-full border

### LINKED ISSUE #5078: Explore sidebar full width borders
ref 
* #4906 

In order to make sidbar filter menu start from the left edge of the screen the SidebarFilter needs to be moved outside of the continaer
image

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5078
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [x] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

File: components/explore/filters/PriceFilter.vue [hunk 1]
```diff
@@ -168,4 +171,12 @@ const toggleInputFocused = (): void => {
     box-shadow: 0 0 0 1px theme('k-blue');
   }
 }
+.padding-left {
+  padding-left: $fluid-container-padding;
+}
```
Comment (roiLeo) [inline components/explore/filters/PriceFilter.vue:176]: can we rename this to something more meaningfull?

**Review by roiLeo — COMMENTED** [review]
Nice work!

mobile version is a bit off

Screenshot 2023-03-11 at 20-59-50 Low minting fees and carbonless NFTs KodaDot - NFT Market Explorer


Otherwise lgtm

File: pages/_prefix/explore/items.vue [hunk 2]
```diff
@@ -31,5 +33,26 @@ const components = { Items }
     }
   },
 })
-export default class ExploreItems extends mixins(ExperimentMixin) {}
+export default class ExploreItems extends mixins(ExperimentMixin) {
+  get isSidebarOpen() {
+    return this.$store.getters['preferences/getsidebarFilterCollapse']
```
[the remark this answers is not in the document]
Comment (daiagi) [reply pages/_prefix/explore/items.vue:38]: yes let's merge this one after #5219

File: components/explore/filters/PriceFilter.vue (hunk 1 above)
Comment (daiagi) [reply components/explore/filters/PriceFilter.vue]: renamed to fluid-padding

**Comment by daiagi** [comment]
>mobile version is a bit off

fixed

File: components/explore/filters/PriceFilter.vue [hunk 3]
```diff
@@ -168,4 +174,12 @@ const toggleInputFocused = (): void => {
...
   }
 }
+.fluid-padding {
+  padding-left: $fluid-container-padding;
+}
+.border-bottom {
+  @include ktheme() {
+    border-bottom: 1px solid theme('border-color');
+  }
+}
```
Comment (roiLeo) [inline components/explore/filters/PriceFilter.vue:184]: move this to global stylesheet

**Review by roiLeo — COMMENTED** [review]
could you try to use new `preferences` stores?

File: components/explore/filters/PriceFilter.vue (hunk 3 above)
Comment (daiagi) [reply components/explore/filters/PriceFilter.vue]: done

File: components/explore/filters/StatusFilter.vue [hunk 4]
```diff
@@ -75,3 +81,20 @@ const applyToUrl = (queryCondition: { [key: string]: any }) => {
...
+}
+
+.border-bottom {
+  @include ktheme() {
+    border-bottom: 1px solid theme('tborder-color');
+  }
+}
+.fluid-padding {
+  padding-left: $fluid-container-padding;
+}
```
[the remark this answers is not in the document]
Comment (daiagi) [reply components/explore/filters/StatusFilter.vue]: done

File: components/explore/filters/StatusFilter.vue [hunk 5]
```diff
@@ -75,3 +81,20 @@ const applyToUrl = (queryCondition: { [key: string]: any }) => {
   emit('resetPage')
 }
 </script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.padding-left {
+  padding-left: $fluid-container-padding;
+}
```
[the remark this answers is not in the document]
Comment (daiagi) [reply components/explore/filters/StatusFilter.vue]: yep, removed

**Comment by daiagi** [comment]
> could you try to use new `preferences` stores?

done

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Review by vikiival — APPROVED** [review]
Really nice work! 🤗

**Comment by vikiival** [comment]
pay 44 usd
